set.mm

```diff
@@ -662596,6 +663406,1223 @@ closed under the multiplication ( ' X ' ) of a finite number of
+    climuz.m $e |- ( ph -> M e. ZZ ) $.
+    climuz.z $e |- Z = ( ZZ>= ` M ) $.
+    climuz.f $e |- ( ph -> F : Z --> CC ) $.
+    $( Express the predicate:  The limit of complex number sequence ` F ` is
+       ` A ` , or ` F ` converges to ` A ` .  (Contributed by Glauco
+       Siliprandi, 2-Jan-2022.) $)
+    climuz $p |- ( ph -> ( F ~~> A <-> ( A e. CC /\ A. x e. RR+ E. j e. Z A. k
+       e. ( ZZ>= ` j ) ( abs ` ( ( F ` k ) - A ) ) < x ) ) ) $=
+      ( vl vi cv cfv cmin clt wral nfcv vy cli wbr cc wcel co cabs cuz wrex crp
+      wa climuzlem wb wceq breq2 ralbidv rexbidv fveq2 raleqdv nffv nfov oveq1d
+      nfbr nfv fveq2d breq1d cbvral a1i bitrd cbvrexv cbvralv anbi2i ) AFCUBUCC
+      UDUEZMOZFPZCQUFZUGPZUAOZRUCZMNOZUHPZSZNHUIZUAUJSZUKZVMEOZFPZCQUFZUGPZBOZR
+      UCZEDOZUHPZSZDHUIZBUJSZUKZAUACNMFGHJKLULWEWQUMAWDWPVMWCWOUABUJVRWJUNZWCVQ
+      WJRUCZMWASZNHUIZWOWRWBWTNHWRVSWSMWAVRWJVQRUOUPUQXAWOUMWRWTWNNDHVTWLUNZWTW
+      SMWMSZWNXBWSMWAWMVTWLUHURUSXCWNUMXBWSWKMEWMEVQWJREVPUGEUGTEVOCQEVNFIEVNTU
+      TEQTECTVAUTERTEWJTVCWKMVDVNWFUNZVQWIWJRXDVPWHUGXDVOWGCQVNWFFURVBVEVFVGVHV
+      IVJVHVIVKVLVHVI $.
+  $}
+
+$(
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+                           Inferior limit (lim inf)
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+$)
+
+  $c liminf $.
+
+  $( Extend class notation to include the liminf function. $)
+  clsi $a class liminf $.
+
+  $d x k $.
+  $( Define the inferior limit of an infinite sequence of extended real
+     numbers.  (Contributed by GS, 2-Jan-2022.) $)
+  df-liminf $a |- liminf = ( x e. _V |-> sup ( ran ( k e. RR |-> inf ( ( ( x "
+    ( k [,) +oo ) ) i^i RR* ) , RR* , < ) ) , RR* , < ) ) $.
+
+  ${
+    $d A i k $.  $d B i $.  $d F i j $.  $d F i k $.  $d i ph $.
+    limsuplt2.1 $e |- ( ph -> B C_ RR ) $.
+    limsuplt2.2 $e |- ( ph -> F : B --> RR* ) $.
+    limsuplt2.3 $e |- ( ph -> A e. RR* ) $.
+    $( The defining property of the superior limit.  (Contributed by Glauco
+       Siliprandi, 2-Jan-2022.) $)
+    limsuplt2 $p |- ( ph -> ( ( limsup ` F ) < A <-> E. k e. RR sup ( ( ( F " (
+       k [,) +oo ) ) i^i RR* ) , RR* , < ) < A ) ) $=
+      ( vi vj clt wbr cv cr cpnf cico co cima cxr cin clsp cfv csup cmpt wss wf
+      wrex wcel wb eqid limsuplt syl3anc wa cvv wceq oveq1 imaeq2d ineq1d simpr
+      supeq1d xrltso supex a1i fvmptd3 breq1d rexbidva cbvrexv 3bitrd ) AEUAUBB
+      KLZIMZJNEJMZOPQZRZSTZSKUCZUDZUBZBKLZINUGZEVJOPQZRZSTZSKUCZBKLZINUGZEDMZOP
+      QZRZSTZSKUCZBKLZDNUGZACNUECSEUFBSUHVIVSUIFGHBCIJEVPVPUJZUKULAVRWDINAVJNUH
+      ZUMZVQWCBKWOJVJVOWCNVPUNWMVKVJUOZSVNWBKWPVMWASWPVLVTEVKVJOPUPUQURUTAWNUSW
+      CUNUHWOSWBKVAVBVCVDVEVFWEWLUIAWDWKIDNVJWFUOZWCWJBKWQSWBWIKWQWAWHSWQVTWGEV
+      JWFOPUPUQURUTVEVGVCVH $.
+  $}
+
+  ${
+    $d A w x y z $.  $d B w x y z $.  $d F x $.
+    $( Ordering property of the superior limit function.  (Contributed by
+       Glauco Siliprandi, 2-Jan-2022.) $)
```

Comment:
I've fixed superior --> inferior (see new commit)
If Mario agrees, we can change both comments as you suggested (I'd prefert to do it after the merge, so that I can dedicate a new pull request just to clean up the mathbox and minimize the risk of conflicts, becasue this pending PR is pretty large)